setup.py

```diff
@@ -11,7 +11,7 @@
 from pathlib import Path
 
 REQUIRED_PACKAGES = ["colorcet", "ebrains_drive", "ipympl>0.8.5", "ipywidgets==7.7.2", "mne>=1.0",
-                     "pythreejs", "pyvista>=0.34.0", "tvb-library>=2.5"]
+                     "pythreejs", "pyvista>=0.34.0", "tvb-library>=2.5", "plotly"]
```

Comment:
here we try to order dependencies in alphabetic order, so plotly would be first on this line